components/back-button/back-button.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./back-button.css";
+import SVG from "react-inlinesvg";
+import leftArrow from "../../public/chevron-left.svg";
+
+const BackButton = ({ text }) => {
```

Comment:
add the export statement before defining the variable, like so:
export const BackButton = ({ text }) => {}